### DISCUSSION TITLE
Progress indicator during web client upload

### DISCUSSION THREAD

**Discussion by blindmikesghost** [drakkan/sftpgo#1352] [Ideas]
Hello,

we're using SFTPGo for transfer of (very) large files. It would be really useful if there was some kin of progress indicator during upload, ideally with a list of files currently being uploaded and how far they got. Could you implement something like this?

Thanks

**Comment by zen85** [comment]
since i just wondered the same thing and saw the confused emoji of drakkan:
me and blindmikeghost would love an additional feature where one has feedback in the form of a progress bar to see the status of the upload of the files. this would make this great piece of work even greater becaue right now i just see a spinner while uploading using the webclient. 
i can see how this might be a pain to implement and how it might be out of scope since you should do such things using something like filezilla anyway.

an interessting feature would also be helpful the other way around: when i share files and the user has no feedback on the total filesize he might get confused. it just starts downloading and the user does not know how long it will approximatly take since he cant see anywhere how big the overall file is... he just sees how much he already downloaded. i would think this is an even more useful feature since i cant get my parents to use an sftp client.

other than that i like the small footprint of sftpgo.... made me switch from nextcloud the moment i saw it.  thank you!

**Reply by PeeBeerBeach** [reply]
Same here!
I also think a progress indicator on web upload would be great.
The issue is that users might just reload the page or cancel the upload when it takes long as they expect that the page hangs.

**Reply by drakkan** [reply]
guys, continuing to ask doesn't help, what helps is to start supporting the project you rely on and in this way ensure that it remains alive and maintained. Thank you

**Reply by PeeBeerBeach** [reply]
I was not aware it is not going that well.
This was more meant like giving voices about how to make it better. This is not connected to any expectations.

But of course i understand your request for donating something to the project. I will do that.

**Reply by PeeBeerBeach** [reply]
Just wondering how to do this as private user. Paypal is the way to go?

**Reply by drakkan** [reply]
yes Paypal is an option, as you prefer

**Reply by PeeBeerBeach** [reply]
You have just received a donation. Not a lot but still hope it helps.

**Comment by drakkan** [comment]
> You have just received a donation. Not a lot but still hope it helps.

Thank you
